buildtest/cli/help.py

```diff
@@ -615,3 +613,8 @@ def buildtest_help(command):
         print_stylecheck_help()
     elif command in ["unittests", "test"]:
         print_unittests_help()
+
+
+def buildtest_help_config():
+    if command in ["profiles", "v"]:
+        print_profiles_help()
```

Comment:
No need to add a function here since you are applying the logic from method `buildtest_help`. Which should actually be `buildtest_show `. Your code is out of date, so make sure to fetch the current code!

Reply:
done